authz-openfga-ex/README.md

```diff
@@ -0,0 +1,30 @@
+# Open FGA Experiment 
+
+# Run Openfga server with postgres
+1) Using docker compose: `docker compose up`
+or using docker scripts
+2) Postgresql database `./start-postgresql.sh`
+2) Openfga playground `./start-openfga.sh`
+
+# Run
+```
+export OPENFGA_API_SCHEME="http" 
+export OPENFGA_API_HOST="0.0.0.0:8080" //openfga running on localhost:8001 (playground localhost:3000)
```

Comment:
comment should say 8080, right? not 8001?

Reply:
yes, fixed